qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -725,36 +848,71 @@ def new_quadratic_sequence(
     pre_post_rotation=False,
     **kwargs
 ):
-    """
-    Creates Quadratic Decoupling Sequence.
+    r"""
+    Creates the quadratic sequence.
 
     Parameters
     ----------
     duration : float, optional
-        defaults to None
-        The total duration of the sequence
-    number_outer_offsets : int, optional
-        Number of outer X-pi Pulses. Defaults to None. Not used if number_of_offsets
-        is supplied.
+        Defaults to None.
+        The total duration of the sequence :math:`\tau`.
     number_inner_offsets : int, optional
-        Number of inner Z-pi Pulses. Defaults to None. Not used if number_of_offsets
-        is supplied
+        Number of inner Z-pi Pulses :math:`n_1`. Defaults to None.
+        Not used if number_of_offsets is supplied.
+    number_outer_offsets : int, optional
+        Number of outer X-pi Pulses :math:`n_2`. Defaults to None.
+        Not used if number_of_offsets is supplied.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        Quadratic sequence
+    DynamicDecouplingSequence
+        The quadratic sequence.
+
+    See Also
```

Comment:
Again, I think See Also, Notes, and References might come after "Raises" (https://numpydoc.readthedocs.io/en/latest/format.html#sections)